delfin/leader_election/distributor/task_distributor.py

```diff
@@ -47,7 +48,10 @@ def __call__(self):
                       six.text_type(e))
 
     def distribute_new_job(self, task_id):
```

Comment:
We need to remove def __call__ ad we should have one methos to handle distribute_delete()

Reply:
Yes